c/meterpreter/source/common/remote.c

```diff
@@ -73,6 +73,11 @@ VOID remote_deallocate(Remote * remote)
 		lock_destroy(remote->lock);
 	}
 
+	if (remote->orig_config)

```

Comment:
this if isn't needed, free does it for you. Just free unconditionally.

Reply:
found another implausible check in the same file, will fix with this land.

Reply:
Added bf8950eb

Reply:
Old habits die hard, my friend!